### ISSUE TITLE
Drawing Text

### ISSUE THREAD

**Issue by devedse** [nkast/WebGLxnaProj#3]
Hi Nkast,

I'm very interested in this project (mostly for fun & out of my own interest). I was curious if implementing text rendering is somewhere on your agenda?

Also, do you have discord where I can ask a few questions around the project?

-Devedse

**Comment by devedse** [comment]
After some playing around when adding a SpriteFont to your solution I kept running into this exception:
```
throw new InvalidOperationException(
    "Failed to get default constructor for ContentTypeReader. To work around, add a creation function to ContentTypeReaderManager.AddTypeCreator() " +
    "with the following failed type string: " + originalReaderTypeString, ex);
```

This exception happens when trying to load the following Type:
```
'Microsoft.Xna.Framework.Content.ListReader`1[[Microsoft.Xna.Framework.Rectangle, MonoGame.Framework]]'
```

After some more debugging I found out that the Rectangle class in your version of MonoGame is in a different namespace. When you try to load this instead it works:

```
'Microsoft.Xna.Framework.Content.ListReader`1[[Microsoft.Xna.Framework.Rectangle, Xna.Framework]]'
```

As a quick fix I added some code to `ContentTypeReaderManager.cs`:
```
//System.Diagnostics.Debug.WriteLine(originalReaderTypeString);

// Need to resolve namespace differences
string readerTypeString = originalReaderTypeString;

readerTypeString = PrepareType(readerTypeString);

var l_readerType = Type.GetType(readerTypeString);

VVVVVVVVVVVVVVVVVVVVVVVVVVVVVVV
//Added the code below
if (l_readerType == null)
{
    var readerTypeString2 = readerTypeString.Replace("MonoGame.Framework]]", "Xna.Framework]]");
    l_readerType = Type.GetType(readerTypeString2);
}
//Added the code above
^^^^^^^^^^^^^^^^^^^^^^^^^^^

if (l_readerType != null)
{
```

**Comment by nkast** [comment]
Hi Devedse,

The issue is now fixed in v3.8.9012.10.
Also added a test component, https://github.com/nkast/WebGLxnaProj/commit/516d2ec08f9e37768f0d5024444ec3834bc9f275

Currently compressed Spritefonts/textures are not supported.

There are still things that are not implemented, I would like to 
complete the Reach feature set, and then maybe move to audio 
or improve on content building/loading.
